### PR TITLE
Test.py exit code accurate in all cases (RIPD-1193):

### PR DESCRIPTION
**Pull request by ximinez**
* Also remove some redundant output.

### CODE REVIEW

File: Builds/Test.py [hunk 1]
```diff
@@ -265,6 +269,11 @@ def main():
                 os.remove('.ninja_deps')
                 os.remove('.ninja_log')
 
+    if all_failed:
+        if len(to_build) > 1:
+            print()
+            print('FAILED:', *(':'.join(f) for f in all_failed))
+        exit(1)
```
Comment (seelabs) [inline Builds/Test.py]: Nit: We probably want `sys.exit`, not `exit`. See http://stackoverflow.com/questions/6501121/the-difference-between-exit-and-sys-exit-in-python

**Comment by seelabs** [comment]
:+1: Looks good to me.

File: Builds/Test.py [hunk 2]
```diff
@@ -196,22 +196,21 @@ def run_tests(args):
...
                     break
-            ARGS.verbose and print(*lines, sep='')
 
             if not ARGS.nonpm:
                 print('npm tests for', target)
                 resultcode, lines = shell('npm', ('test', '--rippled=' + executable,))
                 if resultcode:
-                    print('ERROR:\n', *lines, sep='')
+                    if not ARGS.verbose:
+                        print('ERROR:\n', *lines, sep='')
```
Comment (seelabs) [inline Builds/Test.py]: Curious about this change. Shouldn't we print if we _are_ verbose?

Comment (ximinez) [inline Builds/Test.py]: If we are verbose, we've already printed it from within `shell`, so there's no point in repeating the entire output.

File: Builds/Test.py (hunk 1 above)
Comment (ximinez) [inline Builds/Test.py]: Good tip. Thanks.

**Comment by nbougalis** [comment]
:+1:

**Pull request closed without merging**